ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -2834,24 +2834,28 @@ const triggerSet: TriggerSet<Data> = {
         directions: {
           en: '${start} => ${rotation}',
           de: '${start} => ${rotation}',
+          ja: '${start} → ${rotation}',
```

Comment:
```suggestion
          ja: '${start} => ${rotation}',
```

Mmm, I'm not sure it's worth changing this one instance. `git grep "ja:.*=>" | grep -v "ja: (" | wc -l` suggests there's 180 occurrences of `=>` for Japanese, and I think it'd be better to be consistent than to change this one instance.  (If you think we should change all of them, then maybe that's worth a different PR.)

Reply:
In addition, the `=>` is matched in the line below to be replaced as `や` in Japanese. If you're going to change the symbol, not only should you change all the instances for consistency, but it is also required to make the changes to the regex mentioned to make sure it is still functioning.

https://github.com/quisquous/cactbot/blob/31ef788ad883bdfc3108ee4332e99f526a63c9ed/ui/raidboss/popup-text.ts#L1472-L1474